### ISSUE TITLE
feature request: use of optional b2_list_file_names arguments

### ISSUE THREAD

**Issue by VitaliyAdamkov** [erickj/bkblz#2]
At the [man page of b2_list_file_names Backblaze API operation](https://www.backblaze.com/b2/docs/b2_list_file_names.html) there are some useful optional arguments, _prefix_ and _delimiter_.
It would be wonderful if you make them accessible through **Bkblz::V1::ListFileNamesRequest** from your gem.
Thank you again for your work :)

**Comment by VitaliyAdamkov** [comment]
Sure, there is a better way, but for now I'm using this monkey patch:
```module Bkblz
  module V1
    class ListFileNamesRequest < Request
      def initialize bucket, max_file_count = 1000, optional = {}
        @bucket = bucket
        @body = {
          bucket_id:      bucket.bucket_id,
          max_file_count: max_file_count
        }.merge(optional)
      end
    end
  end
end

**Comment by erickj** [comment]
fixed in 0.1.8, sorry this took so long

**Comment by VitaliyAdamkov** [comment]
thank you :+1:
